listenbrainz/db/user_setting.py

```diff
@@ -0,0 +1,125 @@
+import sqlalchemy
+from datetime import datetime, timedelta
+from listenbrainz import db
+from listenbrainz.db.exceptions import DatabaseException
+
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def get_pg_timezone():
+    """ Get list of time zones PostgreSQL supports.
+
+    Returns:
+        list of tuple('zone_name', 'utc_offset')
+    """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT * FROM pg_timezone_names
+            ORDER BY name
+        """))
+        timezones = [(row["name"], row["utc_offset"]) for row in result.fetchall()]
+        timezones = standardize_timezone(timezones)
+        return timezones
+
+
+def get(user_id: int):
+    """Get user setting with the row ID of the user in the DB, or create if there's setting history.
+    Args:
+        user_id (int): the row ID of the user in the DB
+    Returns:
+        zone_name (str): the user selected timezone name. If not selected by user, return 'UTC'
```

Comment:
This does not reflect what the code does -- if there is no user setting right now it will return None. I think we ought to define a constant in this module called 'DEFAULT_TIMEZONE "UTC"' and then return that constant when no row exists.